src/stores/content.ts

```diff
@@ -99,7 +99,7 @@ export const useContentStore = defineStore('content', () => {
 
   const updateFileTreeTask = useTask(function* () {
     if (isFirstRun.value) {
-      yield updateFileTree().then(() => expandAll())
+      yield updateFileTree().then(() => (expandContent.value ? expandAll() : collapseAll()))
```

Comment:
No need to collapse again
```suggestion
      yield updateFileTree().then(() => (expandContent.value && expandAll()))
```

Reply:
Done